src/Parser.js

```diff
@@ -23,6 +23,7 @@ class Parser {
     this.registerExtension('scss', './assets/SASSAsset');
 
     this.registerExtension('html', './assets/HTMLAsset');
+    this.registerExtension('coffee', './assets/CoffeeScriptAsset');
```

Comment:
Probably better to group this up with JS/TS

Reply:
Done.